@navikt/core/css/form/search.css

```diff
@@ -84,13 +115,13 @@
   min-width: 40px;
 }
 
-.navds-search__button-search.navds-button--tertiary {
+.navds-search__button-search.navds-button--secondary {
```

Comment:
Ref pr møtet, enten sette background: transparent eller bruke tertiary

Reply:
Jeg er usikker på om tertiary blir riktig, for dette ser veldig rart ut:
![image](https://user-images.githubusercontent.com/1894119/160821575-9776383b-6452-4289-9dcc-3d6e7c20526d.png)

Reply:
Kjører på med `transparent` da 🚗

Reply:
Ikke `transparent`, mener du?

Reply:
Det er `transparent` jeg synes ser rart ut, så jeg foreslår at vi beholder bagrunnen.

Reply:
Ah skjønner, ja ser det nå siden textfield ikke er transparent. Lar det bare stå nå da 🚀